TrangleTest.java

```diff
@@ -16,5 +16,8 @@ public void setup() {
 		public void test() {
 			assertEquals(t.Tritype(0, 0, 0), 3);
 		}
-
+	
+		public void test2() {
+			assertEquals(t.Tritype(2, 3, 4), 4);
```

Comment:
Are you sure that the expected answer is correct

Reply:
Yes. I am quite sure. Thanks.

Reply:
Ok. It can be committed.

Reply:
You may close the pull request.